docs/content/en/docs/reference/vsphere/customize-ovas.md

```diff
@@ -97,21 +97,36 @@ Once the VM is powered on and fetches an IP address, ssh into the VM using your
 ssh -i <private-key-file> username@<VM-IP>
 ```
 
-Make desired config changes on the VM 
+Make desired config changes on the VM
+
+### Reset the machine-id and power off the VM
+
+This step in needed because of a [known issue in Ubuntu](https://kb.vmware.com/s/article/82229) which results in the clone VMs getting the same DHCP IP
```

Comment:
```suggestion
This step is needed because of a [known issue in Ubuntu](https://kb.vmware.com/s/article/82229) which results in the clone VMs getting the same DHCP IP
```